pip/requirements.txt

```diff
@@ -26,7 +26,8 @@ requests-toolbelt==0.10.1
 rfc3986==2.0.0
 rich==13.2.0
 secretstorage==3.3.3
-setuptools==66.1.1
+### DO NOT UPGRADE - setuptools > 65.x.x removes flexible versioning ###
+setuptools==65.0.0
```

Comment:
We can't ugprade to 66.0 because it removes non-semantic versioning support - https://github.com/pypa/setuptools/issues/3772#issuecomment-1384342813